web-client/src/presenter/computeds/selectDocumentTypeHelper.test.ts

```diff
@@ -48,16 +49,28 @@ describe('selectDocumentTypeHelper', () => {
         scenario: 'Nonstandard A',
       };
 
+      const mockDocuments = MOCK_DOCUMENTS.map(d => {
```

Comment:
Could this variable be extracted somehow so that it doesn't have to be defined repeatedly in multiple test cases?